tools/installtest.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+#
+# list packages that are not required by packages in this repo
+
+# with-test only looks at test dependencies of the packages listed on the
+# cmdline, not everything recursively
+# so find out recursive deps of xs-toolstack: we want to run tests for all of
```

Comment:
We might want to exclude some packages from being tested, mainly binaries used for development (ocamlformat, merlin, utop and co)